librecad/src/ui/forms/qg_commandwidget.cpp

```diff
@@ -155,7 +157,48 @@ void QG_CommandWidget::tabPressed() {
             leCommand->setText(reducedChoice.first());
         }
         else if (reducedChoice.count()>0) {
+            QString longestString = "";
+            QString shortestString = "";
```

Comment:
It's still true, unless you know there's a default ctor for type. Otherwise, it's undefined behavior for primitive types, the same as in C, or a compiling time error (no default ctor).

you don't have to make a new pull request, as you can just keep pushing your commits to this branch.